src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs

```diff
@@ -33,21 +33,26 @@ public TargetInfo(OSPlatform? os, Architecture? arch, Version targetFrameworkVer
         {
             OS = os ?? HostOS;
             Arch = arch ?? RuntimeInformation.OSArchitecture;
-            FrameworkVersion = targetFrameworkVersion ?? net60;
+            FrameworkVersion = targetFrameworkVersion ?? new Version(7, 0);
 
             Debug.Assert(IsLinux || IsOSX || IsWindows);
 
-            if (FrameworkVersion.CompareTo(net60) >= 0)
+            if (FrameworkVersion.Major == 7)
+            {
+                BundleMajorVersion = 7u;
+                DefaultOptions = BundleOptions.None;
+            }
+            else if (FrameworkVersion.Major == 6)
```

Comment:
Looks like we need to specify 6u for 7 as well:
```suggestion
            if (FrameworkVersion.Major >= 6)
```